### PR TITLE
Fix for maximum possible conversions

### PR DESCRIPTION
**Pull request by BlackFlash5**

### CODE REVIEW

**Comment by BoGuu** [comment]
Care to explain which bit wasn't working?

**Comment by BlackFlash5** [comment]
If you processed something, it checks your free inventory space (line 76) and filled it up with the processed item (109 & 135).
It also doen't delete the unprocessed item, cause of the wrong multiplicator (105 & 131).

This checks based on the materials and free space you have, how much you can converse.

**Comment by BoGuu** [comment]
I made this, Line 105 is removing items. I'll test now

**Comment by BlackFlash5** [comment]
Sorry, i think i found my failure...
I was processing emtpy water bottles to filled up water bottels, what means the unprocessed item is litghter then the processed one. So its a bit different in all these checks. :/

**Comment by BoGuu** [comment]
That's exactly what the checks were put in place for, as the previous Author did not consider it. I'm testing now, hang tight

**Comment by BoGuu** [comment]
Yep there is a problem, this will fix it:

`    private _estConversions = floor(_freeSpace / _netChange);
    if (_estConversions < _minimumConversions) then {
        _minimumConversions = _estConversions;
    };`

File: life_client/core/actions/fn_processAction.sqf [hunk 1]
```diff
@@ -73,11 +73,17 @@ if (_newItemWeight > _oldItemWeight) then {
     _netChange = _newItemWeight - _oldItemWeight;
     _freeSpace = life_maxWeight - life_carryWeight;
     if (_freeSpace < _netChange) exitWith {_exit = true;};
-    _minimumConversions = floor(_freeSpace / _netChange);
+    _maximumConversions = floor(_freeSpace / _netChange);
 };
 
 if (_exit) exitWith {hint localize "STR_Process_Weight"; life_is_processing = false; life_action_inUse = false;};
 
+if(_minimumConversions > _maximumConversions) then {
```
Comment (CamaroZ) [inline life_client/core/actions/fn_processAction.sqf]: if (

**Comment by BlackFlash5** [comment]
Thanks, i'll give it a shot. :)

**Comment by BlackFlash5** [comment]
Works like a charm. Thanks!

**Comment by BoGuu** [comment]
If you chuck a pull request in with the fix I gave you, can merge it

**Pull request closed without merging**